### PR TITLE
Fixed confusing message and install button in version list

### LINKED ISSUE #3229: Install button in version list shows confusing "works with" message
STR:
- Pick an app that is not compatible with Android
- On desktop, open its version page, using android as the app in the URL, like this: https://addons.mozilla.org/en-us/android/addon/fireftp/versions/
- Hover the versions in the list

Expected results:
- Since you're using a desktop browser, the install buttons should not say anything about Android. Or maybe you should have been redirected ? Not sure. Anything but the behavior described below :)

Actual results:
- When hovered, the versions show a green install button with a super confusing warning saying "works in Android" - which is not true - with a "no" red sign.

It looks like it's done by https://github.com/mozilla/addons-server/blob/master/static/js/zamboni/buttons.js ; not sure, lots of duplication between different templates and that js code :(

### PR DESCRIPTION
**Pull request by atiqueahmedziad**
Fixes #3229 
@diox @tofumatt will you please check, if the patch is okey.

### CODE REVIEW

File: static/css/impala/buttons.less [hunk 1]
```diff
@@ -153,6 +153,9 @@ button.bad, .button.bad, .button.developer, .button.scary { // Red
                 }
             }
         }
+        &.download-button-red {
+          background-color: #a91300 !important;
```
Comment (tofumatt) [inline static/css/impala/buttons.less]: Why use `!important` here? Could you make the selector more specific instead?

Comment (tofumatt) [inline static/css/impala/buttons.less]: (It might make more sense to put in `restyle.less`, not sure.

File: static/js/zamboni/buttons.js [hunk 2]
```diff
@@ -404,6 +406,13 @@ var installButton = function() {
         $button.text(gettext('Only with Firefox \u2014 Get Firefox Now!'));
         $button.attr('href', 'https://www.mozilla.org/firefox/new/?scene=2&utm_source=addons.mozilla.org&utm_medium=referral&utm_campaign=non-fx-button#download-fx');
         $('#site-nonfx').hide();
+    } else if (z.app == 'android' && !is_d2c) {
+        $('#downloadAnyway').attr('href',escape_($button.filter(':visible').attr('href')));
+        $('#downloadAnyway').show();
+        $button.addClass('download-button-red');
+        $button.text(gettext('Download now'));
+        $('.extra').hide();
+        addWarning(gettext('Only available with firefox desktop'));
```
Comment (tofumatt) [inline static/js/zamboni/buttons.js]: This should be "Only available on Firefox Desktop" I think.

**Comment by tofumatt** [comment]
Can you also attach before/after screenshots so QA will know what to test?

**Comment by atiqueahmedziad** [comment]
### Before
screenshot from 2016-09-06 18 59 59

### After
screenshot from 2016-09-06 16 13 11

**Comment by atiqueahmedziad** [comment]
@tofumatt done. Please check now :)

**Comment by tofumatt** [comment]
Awesome, looks great! 👍  r+

**Pull request merged**